src/gwc/services/pom.xml

```diff
@@ -21,5 +21,9 @@
       <groupId>org.geoserver</groupId>
       <artifactId>gs-gwc-rest</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.geoserver.cloud</groupId>
+      <artifactId>gs-cloud-starter-webmvc</artifactId>
+    </dependency>
```

Comment:
Cannot remember if this is still required either ... I need to check

Reply:
needed by the VirtualServiceVerifier (which checks if the workspace actually exist before calling the dispatcher).